test/python/circuit/test_circuit_operations.py

```diff
@@ -432,8 +437,10 @@ def test_copy_variables(self):
         copied.add_var(d, 0xFF)
         self.assertEqual({a, b}, set(copied.iter_input_vars()))
         self.assertEqual({c, d}, set(copied.iter_declared_vars()))
+        self.assertEqual({e}, set(copied.iter_declared_stretches()))
         self.assertEqual({a}, set(qc.iter_input_vars()))
         self.assertEqual({c}, set(qc.iter_declared_vars()))
+        self.assertEqual({e}, set(qc.iter_declared_stretches()))
 
```

Comment:
iirc, the spirit of this test is to `copied.add_stretch(f)` and to assert that `{e, f}` is in `copied`, while `qc` is still `{e}` (to catch the case of a shared reference to a Python object).

That said, we then seem to repeat functionally the same test another 20ish times, so probably not a big deal lol